apps/masterbots.ai/lib/hooks/use-mb-chat.ts

```
@@ -100,74 +118,63 @@ export function useMBChat(config?: MBChatHookConfig): MBChatHookCallback {
     },
     async onResponse(response) {
       if (response.status >= 400) {
-        customSonner({ type: 'error', text: response.statusText })
+        toast.error(response.statusText)
 
         if (isNewChat) {
           await deleteThread({
             threadId: params?.threadId ?? activeThread?.threadId,
-            jwt: session?.user?.hasuraJwt,
-            userId: session?.user.id,
+            jwt: session!.user?.hasuraJwt,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)